cmd/ddev/cmd/describe.go

```diff
@@ -48,6 +49,15 @@ running 'ddev describe <projectname>'.`,
 			util.Failed("Failed to describe %s: %v", app.Name, err)
 		}
 
+		// Ensure we have all services to describe for never-started projects.
+		if !fileutil.FileExists(app.DockerComposeFullRenderedYAMLPath()) {
+			_ = app.DockerEnv()
+			err = app.WriteDockerComposeYAML()
+			if err != nil {
+				util.Failed("Failed to run `docker-compose config` for '%s': %v", app.Name, err)
+			}
+		}
```

Comment:
I added a condition here because PhpStorm DDEV Integration runs `ddev describe -j` every X seconds.

Running `docker-compose config` inside `app.WriteDockerComposeYAML()` every X seconds is not what we want.